docs/api/vi.md

```diff
@@ -516,211 +539,238 @@ globalThis.IntersectionObserver === undefined
+Will call next available timer and wait until it's resolved if it was set asynchronously. Useful to make assertions between each timer call.
 
-  ```ts
-  setTimeout(() => {
-    console.log(1)
-  }, 100)
-  setTimeout(() => {
-    Promise.resolve().then(() => {
-      console.log(2)
-      setInterval(() => {
-        console.log(3)
-      }, 40)
-    })
-  }, 10)
+```ts
+let i = 0
+setInterval(() => Promise.resolve().then(() => console.log(++i)), 50)
+
+await vi.advanceTimersToNextTimerAsync() // log: 1
+expect(console.log).toHaveBeenCalledWith(1)
 
-  await vi.runOnlyPendingTimersAsync()
-  ```
+await vi.advanceTimersToNextTimerAsync() // log: 2
+await vi.advanceTimersToNextTimerAsync() // log: 3
+```
 
-## vi.setSystemTime
+### vi.getTimerCount
 
-- **Type**: `(date: string | number | Date) => void`
+- **Type:** `() => number`
 
-  Sets current date to the one that was passed. All `Date` calls will return this date.
+Get the number of waiting timers.
 
-  Useful if you need to test anything that depends on the current date - for example [luxon](https://github.com/moment/luxon/) calls inside your code.
+### vi.clearAllTimers
 
-  ```ts
-  const date = new Date(1998, 11, 19)
+Removes all timers that are scheduled to run. These timers will never run in the future.
 
-  vi.useFakeTimers()
-  vi.setSystemTime(date)
+### vi.getMockedSystemTime
 
-  expect(Date.now()).toBe(date.valueOf())
+- **Type**: `() => Date | null`
 
-  vi.useRealTimers()
-  ```
+Returns mocked current date that was set using `setSystemTime`. If date is not mocked the method will return `null`.
 
-## vi.setConfig
+### vi.getRealSystemTime
 
-- **Type**: `RuntimeConfig`
+- **Type**: `() => number`
 
-  Updates config for the current test file. You can only affect values that are used, when executing tests.
+When using `vi.useFakeTimers`, `Date.now` calls are mocked. If you need to get real time in milliseconds, you can call this function.
 
-## vi.spyOn
+### vi.runAllTicks
 
-- **Type:** `<T, K extends keyof T>(object: T, method: K, accessType?: 'get' | 'set') => MockInstance`
+- **Type:** `() => Vitest`
 
-  Creates a spy on a method or getter/setter of an object.
+Calls every microtask that was queued by `process.nextTick`. This will also run all microtasks scheduled by themselves.
 
-  ```ts
-  let apples = 0
-  const cart = {
-    getApples: () => 13,
-  }
+### vi.runAllTimers
 
-  const spy = vi.spyOn(cart, 'getApples').mockImplementation(() => apples)
-  apples = 1
+- **Type:** `() => Vitest`
 
-  expect(cart.getApples()).toBe(1)
+This method will invoke every initiated timer until the timers queue is empty. It means that every timer called during `runAllTimers` will be fired. If you have an infinite interval, it will throw after 10 000 tries (can be configured with [`fakeTimers.loopLimit`](/config/#faketimers-looplimit)).
```

Comment:
```suggestion
This method will invoke every initiated timer until the timer queue is empty. It means that every timer called during `runAllTimers` will be fired. If you have an infinite interval, it will throw after 10 000 tries (can be configured with [`fakeTimers.loopLimit`](/config/#faketimers-looplimit)).
```

Reply:
@sheremet-va do you mean every 10,000 tries here "it will throw after 10 000 tries"?

Reply:
We are using space because it's an internationally recommended way even though the docs are in English: https://en.wikipedia.org/wiki/Decimal_separator

In Europe "," is used as a decimal separator, so "10,000" is ten, not ten thousand